package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "Sprint-Challenge-RDBMS",
+  "version": "1.0.0",
+  "description": "The purpose of this exercise is to get you used to being quizzed on _interview questions_ commonly asked about Relational Database Management Systems and SQL.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "server": "nodemon"
```

Comment:
What is the function of this line? At least one other person in your cohort uses this but it doesn't seem to do anything. How are you starting your server?